### PR TITLE
Settings - Update bw_settings for mod changes

### PR DESCRIPTION
**Pull request by PabstMirror**
- Settings changes for upcoming mod changes
- Store some settings that were set only on server

### CODE REVIEW

File: addons/settings/BW_Settings.inc.sqf [hunk 1]
```diff
@@ -42,6 +42,8 @@ _settings = [
 
 [QACEGVAR(advanced_fatigue,performanceFactor), 1.4],
 [QACEGVAR(advanced_fatigue,recoveryFactor), 1.6],
+[QACEGVAR(advanced_fatigue,terraingradientfactor), 0.4],
```
Comment (PabstMirror) [inline addons/settings/BW_Settings.inc.sqf:45]: this is new in ace 3.18, it makes walking on slopes and uneven terrain much more exhausting 
IIRC on first mission everyone was getting tired too quickly so we just changed it on the fly to this
I'm just copying what was saved

Comment (kil0byt3) [reply addons/settings/BW_Settings.inc.sqf:45]: was me, during that TVT we were walking directly uphill so i bumped it down pretty big, it'll be worth a re-look later, but these vals are fine for now

File: addons/settings/BW_Settings.inc.sqf [hunk 2]
```diff
@@ -64,6 +66,7 @@ _settings = [
 [QACEGVAR(nightvision,aimDownSightsBlur), 0.25, false, true],
 [QACEGVAR(overheating,enabled), false],
 [QACEGVAR(refuel,rate), 10],
+[QACEGVAR(refuel,progressduration), 0.1],
```
Comment (kil0byt3) [inline addons/settings/BW_Settings.inc.sqf:69]: this might have been for testing, is this the time it takes to attach fuel line to vehicle?

**Review by kil0byt3 — APPROVED** [review]
rest looks good, we might want to server force `[QACEGVAR(finger,maxrange), 7],` as I have seen the mismatch pop up a lot for that one (missions are still often set to maxrange = 3 for some reason)?

File: addons/settings/BW_Settings.inc.sqf (hunk 2 above)
Comment (PabstMirror) [reply addons/settings/BW_Settings.inc.sqf:69]: yeah, I think it was 0 on server,

Comment (kil0byt3) [reply addons/settings/BW_Settings.inc.sqf:69]: yea, should probably not be so low, but the default is slightly too long as well, can we make it like 2 seconds

**Pull request merged**